script.js

```diff
@@ -0,0 +1,75 @@
+
+
+
+// let form= document.getElementsByTagName('form')[0];
+// const submitBtn = document.getElementById('submit_1')
+// console.log({form, submitBtn});
+// form.addEventListener('submit', event =>{
+//     event.preventDefault();
+//     console.log({event});
+//     submitBtn.innerText='Submitted'
+// })
+
+
+let firstname = document.getElementById('firstname');
+
+let lastname = document.getElementById('lastname');
+
+let age = document.getElementById('age');
+
+let email = document.getElementById('email');
+
+let phone = document.getElementById('phone');
+
+let errorMessage = document.getElementById('error_msg');
+
+let butn= document.getElementById('submit_1')
+
+
+// butn.addEventListener('click',()=>{
+//     butn.innerText ="Submitted"
+// });
+
+let form1= document.getElementsByTagName("form")[0];
+form1.addEventListener('submit',()=>{
+    butn.innerText= "Submitted"
```

Comment:
@kratiks26 You already have eventhandel function at line 63, you can move this code to eventhandel function, eventhandel function is also gets called when form submits so we don't need this listener here